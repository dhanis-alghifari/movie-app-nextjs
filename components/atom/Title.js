import React from "react";

function Title({ title }) {
  return (
    <h1 className="text-blue-900 text-xl font-semibold text-center my-10"> {title}</h1>
  );
}

export default Title;
