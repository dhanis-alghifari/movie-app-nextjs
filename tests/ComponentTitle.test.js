import { render } from "@testing-library/react";
import React from "react";
import "@testing-library/jest-dom/extend-expect";
import Title from "../components/atom/Title";

describe("Test if the Title component is working as expected", () => {
    it("<Title /> matches snapshot", () => {
        const component = render(<Title />);
        expect(component.container).toMatchSnapshot();
    });
});