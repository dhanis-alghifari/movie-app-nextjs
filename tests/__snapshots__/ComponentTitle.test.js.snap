// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if the Title component is working as expected <Title /> matches snapshot 1`] = `
<div>
  <h1
    class="text-blue-900 text-xl font-semibold text-center my-10"
  >
     
  </h1>
</div>
`;
